fastlane/lib/fastlane/actions/appledoc.rb

```diff
@@ -71,11 +71,24 @@ def self.run(params)
         appledoc_args = params_hash_to_cli_args(params_hash)
         UI.success("Generating documentation.")
         cli_args = appledoc_args.join(' ')
-        command = "appledoc #{cli_args}".strip + " \"#{params_hash[:input]}\""
+        input_cli_arg = input_param_to_cli_arg(params_hash[:input])
+        command = "appledoc #{cli_args}".strip + " " + input_cli_arg
         UI.verbose(command)
         Actions.sh command
       end
 
+      def self.input_param_to_cli_arg(input_param)
+        cli_arg = ""
+        if input_param.kind_of?(Array)
```

Comment:
Hey @s-hocking, thanks for submitting this PR 👍 

When passing paths to the command line, we should use the built-in methods to do so. So instead of quoting the parameters (which will fail if file names include quotes), we can use `shellescape`. So looking at other fastlane files using this method, this would look something like

```ruby
input_param.map(&:shellescape).join(' ')
```

Since `input_param` can be either a String or an Array, you can use the following syntax:

```ruby
Array(input_param)
```

which should work for both arrays and strings. Let me know if that works 👍

Reply:
Thanks for pointing me in the right direction @KrauseFx !

I'm now using `shellescape` for all input paths. However, all the other parameters that are passed to `appledoc` on the command line are still being quoted.

Is it reasonable to simply `shellescape` all parameters? Or should it be limited to file path parameters only?